pulp_smart_proxy/app/views.py

```diff
@@ -69,5 +69,10 @@ class VersionView(APIView):
         operation_id="version_read",
     )
     def get(self, request):
-        data = {"version": get_plugin_config("smart_proxy").version}
+        data = {
+            "version": get_plugin_config("smart_proxy").version,
```

Comment:
Question is, should this actually be Pulpcore itself? It may be cheating, but that also resolves the version warning (until Foreman bumps to a version number higher than Pulp again).
```suggestion
            "version": get_plugin_config("pulpcore").version,
```

Reply:
there is no "pulpcore" plugin ;)
but you can use `pulpcore.app.apps.PulpAppConfig.version` for that

Reply:
ah, `core` is the right name!
```suggestion
            "version": get_plugin_config("core").version,
```